packages/browser/examples/assets/config/widget/server/sparql_server.json

```diff
@@ -1,4 +1,76 @@
 {
-  "url": "http://localhost:8997/strabon/",
-  "url_parameters": "Query?handle=download&format=SPARQL/JSON&view=HTML&query="
+  "height": 500,
+  "width": 500,
+  "fontSize": 4,
+  "url": "https://demo.liris.cnrs.fr/vcity/geodatadays/strabon/",
+  "url_parameters": "Query?handle=download&format=SPARQL/JSON&view=HTML&query=",
+  "queries": [
+    {
+      "title": "Select Buildings from Version (2012)",
+      "formats": {
+        "graph": "Graph",
+        "table": "Table",
+        "json": "JSON"
+      },
+      "filepath": "./assets/queries/version.rq"
+    },
+    {
+      "title": "Select Building by ID",
+      "formats": {
+        "graph": "Graph",
+        "table": "Table",
+        "json": "JSON"
+      },
+      "filepath": "./assets/queries/buildingByIDQuery.rq"
+    },
+    {
+      "title": "Select Slabs from Building",
+      "formats": {
+        "graph": "Graph",
+        "table": "Table",
+        "json": "JSON"
+      },
+      "filepath": "./assets/queries/ifcSlabQuery.rq"
+    },
+    {
+      "title": "Count Slabs in Building",
+      "formats": {
+        "table": "Table",
+        "json": "JSON"
+      },
+      "filepath": "./assets/queries/ifcSlabCountQuery.rq"
+    }
+  ],
+  "prefixes": {
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
```

Comment:
look duplicated with PREFIX in assets/config/queries/*.rq files
these information should occured one time

Reply:
These prefixes are not necessarily duplicates of those found in the queries. In SPARQL it could be valid to declare different prefixes for the same URI between different queries as long as each query is internally consistent.

The prefixes declared in the config file would be those used by UD-Viz and the prefixes declared in each query may differ.